.github/workflows/run-test.yml

```
@@ -2,9 +2,11 @@ name: Run Test with Gradle
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main" , "feature/**" , "revise/**" ]
```

main だけでよいかと。
```suggestion
    branches: [ "main"]
```
https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows#pull_request
> pull_request イベントのアクティビティの種類が opened、synchronize、または reopened の場合にのみ実行されます。

pull_request イベントの種類の内、 synchronize は PR に対して push したときにトリガーされます。

main マージ時に push イベントが発火されるので on.push.branches は main のみ指定すればよいかと 🆗

---

対応しました。
083ce49ed0edc6a51fe47f38e35a127270e9f8a7

---

pull_request のbranches 指定を削除しましました。
aa2a1a3d7ce95af82b9dff99729924f4c6294bdb